INSTALL.rst

```diff
@@ -150,7 +150,10 @@ etc., you can install the following:
 * `ImageMagick <https://www.imagemagick.org/script/index.php>`_: for saving
   animated gifs;
 * `LaTeX <https://miktex.org/>`_ and `GhostScript (>=9.0)
-  <https://ghostscript.com/download/>`_ : for rendering text with LaTeX.
+  <https://ghostscript.com/download/>`_ : for rendering text with LaTeX;
+* fontconfig_ (>= 2.7): for detection of system fonts on Linux.
+
+.. _fontconfig: https://www.fontconfig.org
```

Comment:
Is there a particular reason for using a different linking syntax for this here rather than the rest of the file?

Reply:
switched it back to be consistent with the rest